### PR TITLE
Create simple HTTP-server

### PR DESCRIPTION
**Pull request by Mikhailin98**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import (
+	"fmt"
+	"net/http"
+)
+
+func hello(w http.ResponseWriter, req *http.Request) {
+	fmt.Fprintf(w, "hello\n")
+}
+
+func headers(w http.ResponseWriter, req *http.Request) {
```
Comment (vsysoev) [inline main.go:12]: Remove headers

File: main.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+)
+
+func hello(w http.ResponseWriter, req *http.Request) {
+	fmt.Fprintf(w, "hello\n")
```
Comment (vsysoev) [inline main.go:9]: Reply just with time.Now().String()

File: main.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+func headers(w http.ResponseWriter, req *http.Request) {
+	for name, headers := range req.Header {
+		for _, h := range headers {
+			fmt.Fprintf(w, "%v: %v\n", name, h)
+		}
+	}
+}
+
+func main() {
+	http.HandleFunc("/hello", hello)
```
Comment (vsysoev) [inline main.go:21]: Just / handler.

File: main.go [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+	http.HandleFunc("/headers", headers)
+
+	http.ListenAndServe(":8090", nil)
```
Comment (vsysoev) [inline main.go:24]: URL to listen to as command line parameter

**Review by vsysoev — CHANGES_REQUESTED** [review]

File: main.go (hunk 3 above)
Comment (CriptoMikael) [reply main.go]: Edited.

File: main.go (hunk 4 above)
Comment (CriptoMikael) [reply main.go]: Edited.

File: main.go (hunk 2 above)
Comment (CriptoMikael) [reply main.go]: Changed fmt.Fprintf to work with a channel

File: main.go (hunk 1 above)
Comment (CriptoMikael) [reply main.go]: Done.

**Review by vsysoev — APPROVED** [review]

**Pull request merged**